### DISCUSSION TITLE
Regarding wse2

### DISCUSSION THREAD

**Discussion by Santoshkumarkaushik** [nguyen-group/QERaman#16] [General]
Dear Sir
I have calculated raman of wse2 . Experiment shows it has two peaks A1 at 252 and E' at 250 but our spectra shows one peak at 252 . How to get E' peak. How to improve spectra sir?

**Comment by nguyen-group** [comment]
Hi,

You can check in ph.out, if there are two phonon modes at 250 and 252, you can plot them by changing the value of gamma or gamma_raman. 

Best,
Nguyen
